package.json

```diff
@@ -46,8 +46,7 @@
     "*.js": "eslint --cache --fix"
   },
   "resolutions": {
-    "ember-cli-htmlbars/semver": "~7.3.0",
-    "ip-regex": "^2.1.0"
+    "@embroider/util": "^1.0.0"
```

Comment:
Had to add this, to fix he Embroider scenarios, which are now running with Ember 4.1. See https://github.com/tildeio/ember-element-helper/pull/73#pullrequestreview-866490304

@jelhan maybe we need to postpone the release a bit, until a new version of element-helper is released, to not cause the same issue for users?

Reply:
I guess postponing the stable release might be good in that case. I fear it would otherwise give users of Ember Bootstrap the wrong impression that the release is broken and relying on Embroider is causing too much issues.

Maybe we could do another pre-release in the meantime and announce that this pre-release will very likely be the final one?

Reply:
@jelhan done: https://github.com/kaliber5/ember-bootstrap/releases/tag/v5.0.0-rc.0